packages/management/framework/bundle/config/packageinstalls/tanzu-auth-pi.yaml

```diff
@@ -1,38 +0,0 @@
-#@ load("@ytt:yaml", "yaml")
-#@ load("@ytt:data", "data")
```

Comment:
can you provide some rationale for this deletion and if it is temporary.

Reply:
Removed this commit. This might got fixed in the `main` branch.